src/utils/index.ts

```diff
@@ -18,6 +18,16 @@ export const uint32Le = (uint32: string) => {
   return `0x${[...new Uint8Array(buf)].map(v => v.toString(16).padStart(2, '0')).join('')}`
 }
 
+export const littleHexToInt = (littleHex: string) => {
+  assertIsHexStr(littleHex)
```

Comment:
It's possible to use https://github.com/Keith-CY/molecule-javascript/pull/8/files#diff-2cfb1554a9cbe2bf7981031a26f82e4eR41 here.

Reply:
Updated. https://github.com/Keith-CY/molecule-javascript/pull/8/commits/c4361b5cfa7ac6b3849a29b3dd7c81410ee33a35